src/KafkaW.cxx

```diff
@@ -33,6 +35,7 @@ BrokerOpt::BrokerOpt() {
       {"coordinator.query.interval.ms", 2 * 1000},
       {"heartbeat.interval.ms", 500},
       {"statistics.interval.ms", 600 * 1000},
+      //{"message.timeout.ms",                            6000},
```

Comment:
Yes, no longer needed.
Addressed in #7.
At some point, also the uncommented settings could go away and into the configuration file, but I would address that issue later because it would require a update to the configuration file of running systems.